packages/web/docs/src/pages/docs/management/targets.mdx

```diff
@@ -51,15 +51,19 @@ and configure the following settings:
 
 #### Traffic Threshold and Duration
 
-You can choose to mark a breaking change as safe, if it affects some percentage of your traffic, in
-a specific time range.
+You can choose to mark an incompatible change as safe, if it affects less than some amount of your
```

Comment:
This is me being picky.
Correct me if I'm wrong -- but I think "compatibility" is the technical definition for the spec and "breaking" is the notion of how that compatibility impacts real clients.